velox/docs/functions/spark/string.rst

```diff
@@ -20,6 +20,21 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
     If ``n < 0``, the result is an empty string.
     If ``n >= 256``, the result is equivalent to chr(``n % 256``).
 
+.. spark:function:: concat_ws(separator, [string]/[array<string>], ...) -> varchar
+
+   Returns the concatenation for ``string`` & all elements in ``array<string>``, separated by
+   ``separator``. Only accepts constant ``separator``. It takes variable number of remaining
```

Comment:
Does Spark allow non-constant separator?

Reply:
Confirmed, yes. Just updated the code to support this. Thanks!